api/reviewer_api/services/redactionlayerservice.py

```diff
@@ -15,7 +15,7 @@ def getdefaultredactionlayerid(self):
     def getmappedredactionlayers(self, redactionlayer):     
         mpxlayers = []  
         defaultlayerid = self.getdefaultredactionlayerid()
-        if redactionlayer["redactionlayerid"] != defaultlayerid:
+        if redactionlayer["redactionlayerid"] != defaultlayerid and redactionlayer["redactionlayerid"] != 3:
```

Comment:
I needed to change this to isolate the oipc layer annotations after they are copied over, since all of the default layer annotations were showing up as well

Reply:
Changes handled.